data processing/csv_to_json.py

```diff
@@ -0,0 +1,35 @@
+import csv, json, datetime
+
+# Specify the directory path
+csv_file_name = r'C:\Users\camil\Robotics\JoinedData\PNCMP2024-MatchExport-Q100.csv' #TODO:check if file name need changing later
+json_file_name = r'C:\Users\camil\Robotics\JoinedData\DCMPQualMatches.json'
+
+all_matches = []
+total_time = None
+current_time = datetime.datetime.hour*1000 + datetime.datetime.minute*100 + datetime.datetime.second + float(1/datetime.datetime.microsecond)
+
```

Comment:
What are these variables and what do they do?

Reply:
alll_matches is the array that's used to hold all of the data for the JSON files
total_time and current_time is set there to be able to figure out the total time it takes for this code to run

Reply:
There are 3600 seconds in an hour and 60 seconds in a minute.